guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,135 @@
+
+*In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*
+
+> Reference topics provide quick access to fact-based information. In technical information, reference topics are used to list product specifications and parameters, provide essential data, and provide detailed information on subjects such as the commands in a programming language. [- DITA Reference Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-reference-topic.html)
+
+### Troubleshooting topics
+
+Provide a condition that the reader may want to correct, followed by one or more descriptions of its cause and suggested remedies.
+
+It's good practice to use three standard headlines (like “Condition”, “Cause”, “Remedy”), each might have just one sentence or more as needed. For remedy, use a numbered list if there are multiple steps to follow, and a bullet list if there are several equally valid solutions.
+
+> In its simplest form, troubleshooting information follows this pattern:
+>
+> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
+> 2. A cause for the condition or symptom.
+> 3. A remedy for the condition or symptom.
+>
+> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> * Multiple causes with the same remedy
+> * A single cause with more than one remedy
+> * A remedy with no known cause
+> * A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+> [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+*Klaudia is preparing the new diagram style, see <https://github.com/kyma-project/community/issues/542>.*
+
+### Screenshots
+
+Wherever possible, present screenshots as SUI (simplified user interfaces). Basically, this means to blur out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Documentation Structure and Target Groups
```

Comment:
I would combine the documentation structure and target groups with information types that you described above, for example by listing our doc types (overview, getting started, etc.) and adding information about who is the target groups for this doc type and what information type is expected there. I really like the excel sheet you prepared some time ago, but I also know that tables in Markdown are quite inconvenient to use 😕 Let's discuss.

Reply:
I believe information type and target group don't always need to coincide (even though there might be overlaps - more concept topics in the Overview section). 
There might be conceptual information (for example, about profiles) that's too detailed to newbies/decision makers, but is interesting for admins.

Reply:
I could convert the information to a table once we're fairly sure about the content. But I believe editing is going to be a hassle, so we could keep the normal paragraphs while it's work in progress.